frontend/src/components/Home.vue

```diff
@@ -11,7 +11,7 @@ import QuizCard from '@/components/QuizCard.vue';
         </div>
         <!-- Quiz details will be inserted as attributes to QuizCard components
             for each quiz in quizzes list -->
-        <QuizCard v-for="num in [1, 2, 3, 4, 5, 6, 7]" v-bind:index="num" />
+        <QuizCard v-for="num in 7" v-bind:key="num" />
```

Comment:
drive-by change.

- I'm using `key` instead of `index` because `eslint` was throwing an error. See https://eslint.vuejs.org/rules/require-v-for-key.html
- I simplified the `v-for` expression

Reply:
* I'm okay, `key` is more suitable (thanks to `eslint` !).
* An amazing simplification, by the way ! Didn't know that was possible.